tests/test_jmcomic/test_jm_download_manifest.py

```
@@ -0,0 +1,1053 @@
+                return photo
+
+        downloader = Downloader()
+
+        self.assertIs(downloader.download_album(album_id='123'), album)
+        self.assertEqual(album.duration, 2.5)
+        self.assertIs(asyncio.run(downloader.download_photo(photo_id='456')), photo)
+        self.assertEqual(photo.duration, 4.0)
+
+    def test_export_plugins_allow_omitting_downloader(self):
+        _, photo, _ = new_album_photo_images()
+        option = ContractOption('/tmp')
```

Ruff (0.16.1)

[error] Probable insecure usage of temporary file or directory: "/tmp"

(S108)